### PR TITLE
Added resources related to Web Development.

### PR DESCRIPTION
**Pull request by Trupti0406**
Added various resources related to Web Development. Please check my Pull Request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,9 +138,26 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 **[⬆ Back to Index](#content)**
 
+## Back-End Interview Series
+
+- :video_camera: [Backend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiHI7u_6G8j1oKyhQn_plX8i)
```
Comment (shahednasser) [inline README.md:143]: Please add this under NodeJS instead

File: README.md [hunk 2]
```diff
@@ -427,6 +446,22 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 **[⬆ Back to Index](#content)**
 
+## Frontend Interview Series
```
Comment (shahednasser) [inline README.md:449]: Move this to the Web Development category that you created below

File: README.md [hunk 3]
```diff
@@ -427,6 +446,22 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
...
+
+- :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
+
+---
+
+**[⬆ Back to Index](#content)**
+
+## Frontend with HTML CSS JS & Browser APIs
+
+- :video_camera: [Frontend with HTML CSS JS & Browser APIs](https://youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC)
```
Comment (shahednasser) [inline README.md:459]: Move this to the Web Development category that you created below

File: README.md [hunk 4]
```diff
@@ -702,6 +743,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [MySQL Tutorial for Beginners - Programming with Mosh](https://youtu.be/7S_tz1z_5bA)
 - :video_camera: [SQL Tutorial - Full Database Course for Beginners - freeCodeCamp](https://youtu.be/HXV3zeQKqGY)
 - :video_camera: [MySQL Database - Full Course - freeCodeCamp](https://youtu.be/ER8oKX5myE0)
+- :video_camera: [Practical SQL Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFRR58Ezyy2z3gYhPA0lKmt)
```
Comment (shahednasser) [inline README.md:746]: Please add this under a new "Database" category, then place it in the correct alphabetical position and add it into the table of content

File: README.md [hunk 5]
```diff
@@ -952,6 +1006,14 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 **[⬆ Back to Index](#content)**
 
+## Web Development Course
```
Comment (shahednasser) [inline README.md:1009]: Rename this to Web Development and place it in the table of content

File: README.md [hunk 6]
```diff
@@ -138,9 +138,26 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
...
 
+## Back-End Interview Series
+
+- :video_camera: [Backend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiHI7u_6G8j1oKyhQn_plX8i)
+
+---
+
+**[⬆ Back to Index](#content)**
+
+## Big Data
```
Comment (shahednasser) [inline README.md:149]: Please add this to the table of content

File: README.md [hunk 7]
```diff
@@ -717,6 +759,15 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :green_book: [The only NodeJs introduction you’ll ever need](https://codeburst.io/the-only-nodejs-introduction-youll-ever-need-d969a47ef219)
 - :video_camera: [Introduction to Node.js](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH_uZs4vJMIhcinABSTUH2bY)
 - :green_book: [Back End Development and APIs](https://www.freecodecamp.org/learn/back-end-development-and-apis/)
+- :video_camera: [Backend Development with express](https://youtube.com/playlist?list=PL-Jc9J83PIiEnK1q9tuVrrORqKBexcE_J)
+
+---
+
+**[⬆ Back to Index](#content)**
+
+## Operating Systems
```
Comment (shahednasser) [inline README.md:768]: Please add this in the table of content

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @Trupti0406 for your contribution. I have added some required changes below.

File: README.md (hunk 1 above)
Comment (Trupti0406) [reply README.md:143]: Done!

File: README.md (hunk 2 above)
Comment (Trupti0406) [reply README.md:449]: Done!

File: README.md (hunk 3 above)
Comment (Trupti0406) [reply README.md:459]: Done!

File: README.md (hunk 4 above)
Comment (Trupti0406) [reply README.md:746]: Done!

File: README.md (hunk 6 above)
Comment (Trupti0406) [reply README.md:149]: Done!

File: README.md (hunk 7 above)
Comment (Trupti0406) [reply README.md:768]: Done!

**Comment by Trupti0406** [comment]
@shahednasser I've made all the required changes. Please check.

**Comment by shahednasser** [comment]
Hi @Trupti0406 you need to commit the changes first to appear here in the PR

**Pull request closed without merging**